src/anaconda_auth/token.py

```diff
@@ -390,3 +390,9 @@ def delete_repo_token(self, org_name: OrgName) -> None:
         # We need to remove the existing token for this org first
         # TODO: We can drop this once we just use a dictionary instead
         self.repo_tokens[:] = [t for t in self.repo_tokens if t.org_name != org_name]
+
+    def delete_all_repo_token(self) -> None:
+        """Delete all repo tokens"""
+        # TODO: Confirm whether we should raise an exception or not if token doesn't exist for specified organization
+
+        self.repo_tokens[:] = []
```

Comment:
This won't persist on disk unless we save. It probably makes sense to do that outside this method.